portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -42,12 +42,37 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println("Hello Chiamaka");
 
   }
+  
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOException {
+
+    // Get the input from the form.
+    String input = getContent(req);
+
+    ArrayList<String> comments = new ArrayList<String>();
+    comments.add(input);
+
+    // Redirect back to the HTML page.
+    res.setContentType("text/html;");
+    String json = convertToJsonUsingGson(comments);
+    res.getWriter().println(json);
```

Comment:
All of this gets immediately thrown out, because you're redirecting

Reply:
So should I not redirect? It does that in the example.

Reply:
Redirecting is fine, there's just no need to output anything else